src/public/modules/core/services/gtag.client.service.js

```diff
@@ -109,6 +109,18 @@ function GTag(Auth, $rootScope, $window) {
     }
   }
 
+  /**
+   * Logs an admin user's failure to create form
+   * @param  {Object} form The form object
+   * @return {Void}
+   */
+  gtagService.createFormFailed = () => {
+    _gtagEvents('create_form', {
+      event_category: 'Create Form',
+      event_action: 'Create Form failed',
```

Comment:
Do we need to log any (or even have any) additional data?

Reply:
@karrui Thanks! I added user email (perhaps can use for troubleshooting if user gives feedback); apart from that, can't really think of anything else as the form hasn't been created at that point